### PR TITLE
fixes #350 remove post-body margin-left on small screen [2]

### LINKED ISSUE #350: UI: large margin on mobile device

### PR DESCRIPTION
**Pull request by hsw**
Remove blank space bellow avatar on one-column layout

### CODE REVIEW

File: themes/helvetica/app.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @import "sidebar";
 @import "forms";
 @import "post";
-@import "../shared/timeline-post";
+@import "timeline-post";
```
Comment (clbn) [inline themes/helvetica/app.scss]: Why not shared? That way it will be both in Helvetica and Fresh.

**Comment by hsw** [comment]
Update patch for fresh theme.
"helvetica" theme use shared/timeline-post and ignore local timeline-post.
"fresh" theme use local timeline-post and ignore shared.

**Comment by clbn** [comment]
Why do we need all three of them when we already using `shared` and `helvetica/single-post.scss` almost identical to `fresh/single-post.scss`? I'd suggest creating `shared/single-post.scss` and move there everything common, and cute down `helvetica/single-post.scss` and `fresh/single-post.scss` to the only different line.

**Comment by hsw** [comment]
I remove `.timeline-post-container .post-body .text { font-family: }`.
`themes/fresh` should override `body { font-family: Arial, sans-serif; }` globally if necessary.

File: themes/shared/single-post.scss [hunk 2]
```diff
@@ -7,10 +7,12 @@
   .avatar {
     float: left;
     line-height: 0;
+    @media (max-width: 767px) { margin: 0 10px 10px 0; }
```
Comment (clbn) [inline themes/shared/single-post.scss]: I believe `0 10px 5px 0` looks better (along with the change in `.title`, see below). Could you please check it?

File: themes/shared/timeline-post.scss [hunk 3]
```diff
@@ -18,7 +20,6 @@
     }
```
Comment (clbn) [inline themes/shared/timeline-post.scss]: I believe `3px` here would look better (along with the change in `.avatar`).

File: themes/shared/timeline-post.scss [hunk 4]
```diff
@@ -7,10 +7,12 @@
   .avatar {
     float: left;
     line-height: 0;
+    @media (max-width: 767px) { margin: 0 10px 10px 0; }
```
Comment (clbn) [inline themes/shared/timeline-post.scss]: I believe `0 10px 5px 0` looks better (along with the change in `.title`, see below). Could you please check it?

File: themes/shared/single-post.scss [hunk 5]
```diff
@@ -22,9 +24,8 @@
     }
```
Comment (clbn) [inline themes/shared/single-post.scss]: I believe `3px` here would look better (along with the change in `.avatar`).

**Comment by clbn** [comment]
There are some comments on improving margins. Otherwise, looks good.

**Comment by hsw** [comment]
@clbn please, check last version

**Comment by clbn** [comment]
:+1:

**Pull request merged**